site/pages/meet-our-team/test.js

```diff
@@ -0,0 +1,152 @@
+      expect(previousLink.text()).to.equal('Previous page');
+      expect(nextLink.attr('href')).to.equal('/about-us/people/category/everyone/page-2');
+      expect(nextLink.text()).to.equal('Next page');
+    });
+  });
+
+  describe('with 20 Badgers, second page of Everyone', () => {
+    it('should render badgers list with no advert and link to page 1', () => {
+      const stateNavigator = createStateNavigator();
+      stateNavigator.navigate('badgers', { page: 2 }, 'none');
+
+      const badgers = [];
+      for (let i = 0; i < 20; i += 1) {
+        badgers.push({ firstName: 'Alex ' + i });
+      }
+
+      const teamSlice = render(
+        <Layout stateNavigator={stateNavigator}>
+          <MeetOurTeam
+            categories={[]}
+            category="everyone"
+            badgers={badgers}
+            page={2}
+          />
+        </Layout>);
+
+      const badgersList = teamSlice.find('ul').last();
+      const links = teamSlice.find('a');
+      const previousLink = links.slice(-2, -1);
+      const nextLink = links.slice(-1);
+      expect(badgersList).to.match(/Alex 19/);
+      expect(badgersList).to.not.match(/Alex 18/);
+      expect(badgersList).to.not.match(/Are you a potential Badger/);
+      expect(previousLink.attr('href')).to.equal('/about-us/people');
+      expect(previousLink.text()).to.equal('Previous page');
+      expect(nextLink.attr('href')).to.equal(undefined);
+      expect(nextLink.text()).to.equal('Next page');
+    });
+  });
+
+  describe('with 20 Badgers, first page of Engineering', () => {
+    it('should render badgers list with no advert and no link to page 2', () => {
+      const stateNavigator = createStateNavigator();
+      stateNavigator.navigate('badgers', { category: 'engineering' }, 'none');
+
+      const badgers = [];
+      for (let i = 0; i < 20; i += 1) {
+        badgers.push({
+          firstName: 'Alex ' + i,
+          categories: [
+            { name: 'Engineering', slug: 'engineering' },
+          ],
+        });
+      }
+
+      const teamSlice = render(
+        <Layout stateNavigator={stateNavigator}>
+          <MeetOurTeam
+            categories={[{ name: 'Engineering', slug: 'engineering' }]}
+            category="engineering"
+            badgers={badgers}
+            page={1}
+          />
+        </Layout>);
+
+      const badgersList = teamSlice.find('ul').last();
+      const links = teamSlice.find('a');
+      const previousLink = links.slice(-2, -1);
+      const nextLink = links.slice(-1);
+      expect(badgersList).to.match(/Alex 0/);
+      expect(badgersList).to.match(/Alex 18/);
+      expect(badgersList).to.match(/Alex 19/);
+      expect(badgersList).to.not.match(/Are you a potential Badger/);
+      expect(previousLink.attr('href')).to.equal(undefined);
+      expect(previousLink.text()).to.equal('Previous page');
+      expect(nextLink.attr('href')).to.equal(undefined);
+      expect(nextLink.text()).to.equal('Next page');
+    });
+  });
+});
```

Comment:
I think some of these tests are starting to look a bit over-specified. Might be worth a chat to decide if this is too detailed. Fine for this PR though!